go/teams/teams.go

```diff
@@ -384,6 +384,34 @@ func (t *Team) AllApplicationKeys(ctx context.Context, application keybase1.Team
 	return AllApplicationKeys(t.MetaContext(ctx), t.Data, application, t.chain().GetLatestGeneration())
 }
 
+func (t *Team) AllApplicationKeysWithKBFS(ctx context.Context, application keybase1.TeamApplication) (res []keybase1.TeamApplicationKey, err error) {
+	teamKeys, err := t.AllApplicationKeys(ctx, application)
+	if err != nil {
+		return res, err
+	}
+	kbfsKeys := t.KBFSCryptKeys(ctx, application)
+	if len(kbfsKeys) > 0 {
+		latestKBFSGen := kbfsKeys[len(kbfsKeys)-1].Generation()
+		for _, k := range kbfsKeys {
+			res = append(res, keybase1.TeamApplicationKey{
+				Application:   application,
+				KeyGeneration: keybase1.PerTeamKeyGeneration(k.KeyGeneration),
+				Key:           k.Key,
+			})
+		}
+		for _, tk := range teamKeys {
+			res = append(res, keybase1.TeamApplicationKey{
+				Application:   application,
+				KeyGeneration: keybase1.PerTeamKeyGeneration(tk.Generation() + latestKBFSGen),
```

Comment:
Are you sure this isn't off-by-one?  My formula last night had a `-1` or `+1`, I forget...